api/v1beta1/agentserviceconfig_types.go

```diff
@@ -82,6 +89,10 @@ type AgentServiceConfigSpec struct {
 	// that are used if one the operators fails to be successfully deployed
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Must-Gather Images"
 	MustGatherImages []MustGatherImage `json:"mustGatherImages,omitempty"`
+
+	// ImageServiceOptions exposes variables for the image service that can be set
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Image Service Options"
+	ImageServiceOptions *ImageServiceOptions `json:"imageServiceOptions,omitempty"`
```

Comment:
nit: IMO we don't need "Options" in the field name. This is a section within the Spec, and I don't think we need each section to re-declare itself as `FooSpec`, `FooOptions`, `FooConfig`, etc. Just `Spec.Foo` will do.